deployment/legacy-migration/maintenance-worker/migration/neo4j/import.sh

```diff
@@ -1,17 +1,48 @@
 #!/usr/bin/env bash
 set -e
 
-SECONDS=0
-SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-
-echo "MATCH (n) DETACH DELETE n;" | cypher-shell
+# import .env config
+set -o allexport
+source $(dirname "$0")/.env
+set +o allexport
 
-for collection in "badges" "categories" "users" "follows" "contributions" "shouts" "comments"
-do
-  for chunk in /tmp/mongo-export/splits/$collection/*
+# Import collection function defintion
+function import_collection () {
+  for chunk in ${IMPORT_PATH}splits/$1/*
   do
-    mv $chunk /tmp/mongo-export/splits/current-chunk.json
-    echo "Import ${chunk}" && cypher-shell < $SCRIPT_DIRECTORY/$collection.cql
+    mv $chunk ${IMPORT_CHUNK_PATH}
+    NEO4J_COMMAND="$(envsubst '${IMPORT_CHUNK_PATH_CQL}' < $(dirname "$0")/$1.cql)"
```

Comment:
Or is `envsubst` the command to replace `$VARIABLE` in arbitrary files?

Reply:
yes it is